standards/alerting.md

```diff
@@ -6,6 +6,8 @@ Multiple alarms MAY be set up.
 
 Owners SHOULD identify and monitor other key success metrics.
 
+Alarms SHOULD be tested as part of the [Production Readiness](production-readiness.md) process.
```

Comment:
Might be nice to say how they would be tested. E.g., turning off upstream service, or reducing or raising alarm threshold in order to artificially trigger alarm. Verify all people get emails that should; and that dashboard updates appropriately.

Reply:
That's a good point.

Come to think of it, this might require a bigger discussion and probably shouldn't be included in this post-morterm. I'm going to remove this edit from this and open a PR to the Alerts standard for more discussion.